database_admin/migrations/129_create_pg_repack.up.sql

```diff
@@ -0,0 +1 @@
+CREATE EXTENSION IF NOT EXISTS pg_repack;
```

Comment:
DB migration fails, pg_repack extension is not in the DB image used in ephemeral env.
```
Error upgrading the database: migration failed: extension "pg_repack" is not available, Could not open extension control file "/usr/share/pgsql/extension/pg_repack.control": No such file or directory. in line 0: CREATE EXTENSION IF NOT EXISTS pg_repack;
```
We could change the schema migration to try to find the extension in https://www.postgresql.org/docs/current/view-pg-available-extensions.html and run CREATE EXTENSION... only if the pg_repack extension is available

Reply:
Fixed